components/scg/src/scg-scene.js

```diff
@@ -198,18 +198,22 @@ SCg.Scene.prototype = {
 
     /**
      * Remove object from scene.
-     * @param {SCg.ModelObject} obj Object to remove
+     * @param obj Object to remove
      */
     removeObject: function (obj) {
-        var self = this;
+        let self = this;
```

Comment:
i've added 
`struct =
[*
	some_reflexive_set -> some_reflexive_node;;
	some_reflexive_node => nrel_itself: some_reflexive_node;;
*];;`
a couple of times to kb and then opened struct in distanced-based mode in one tab, in second tab I opened some_reflexive_node and removed all access edges from struct to this node and first tab showed incorrect scg, maybe because of copies processing
![image](https://github.com/ostis-ai/sc-web/assets/56753138/fe850761-7b30-4daf-b4af-db0f8da2cb29)
![image](https://github.com/ostis-ai/sc-web/assets/56753138/f15e0a42-e707-476e-a764-053938426910)

Reply:
fixed